src/overlays/actors/ovl_Obj_Driftice/z_obj_driftice.c

```diff
@@ -195,12 +194,8 @@ void func_80A66930(ObjDrifticeStruct2* arg0, ObjDriftice* this, s16* arg2, s16*
 
     temp_f22 *= arg0->unk_08;
 
-    *arg2 = (s32)temp_f22 + arg0->unk_00[2];
+    *arg2 = arg0->unk_00[2] + (s32)((void)0, temp_f22);
```

Comment:
I tried a few things, `(void)0` seems to suggest there's another temp somewhere and you can add variables to the stack without shifting the stack. Yet nothing I tried seems to work. There should be a simple solution here somewhere but...

Reply:
There was something where I had a (void)0 originally in OoT's EnZf, and EnWf and EnGeldB, and it turned out the temps could be rewritten quite differently to get it to work without (of course it hasn't been subjected to -g3 yet...).

Either way, if something is obviously fake like this, I think it is worth noting with `//! Fake match` or similar, if someone wants to search for fake matches to improve later.

Reply:
I agree about the fake match label. It would be nice to be able to easily find where the fake matches are

Reply:
+1 for a fake match label. Personally I would vote for `//! FAKE:` to mirror the syntax of `//! TODO:`. Atleast for vscode, this would also auto color the label like `TODO`'s. Currently I've just been using the phrase `POSSIBLE FAKE MATCH`.

Reply:
I've gone and added `//! FAKE:` to a few places in the repo

Reply:
Does the colon actually do anything? It's possible that Doxygen will interpret it, but if we're not leaving explanatory comments afterwards, it does look a bit odd.

Reply:
Admittedly not really a big deal, but a colon causes a Vscode to highlight it similar to a `TODO:`. 
I also think it wouldn't be a bad idea to leave a little comment explain the fake match, though most are probably going to be pretty obvious.

Reply:
I have no preference with or without the colon. I'll let you guys decide. I'll leave it for now, but let me know if I should change it.